src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,17 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int horsePowers;
+    private String type;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(int horsePowers, String type) {
+        this.horsePowers = horsePowers;
+        this.type = type;
+    }
```

Comment:
Do you use this constructor/model specific fields anywhere? I guess we can use them when producing objects